core/trino-main/src/main/java/io/trino/server/ThrowableMapper.java

```diff
@@ -93,8 +85,6 @@ public Response toResponse(Throwable throwable)
                     .build();
             case WebApplicationException webApplicationException -> webApplicationException.getResponse();
             default -> {
-                log.warn(throwable, "Request failed for %s", request.getRequestURI());
```

Comment:
> "Request could be already recycled"

Is that really valid? Was the log message not useful?

Reply:
Yes. The ThrowableMapper can be called when the response write has failed - in this case, mapping of the exception to the response doesn't make sense at all as you won't be able to write it anyway